fhir-database-utils/src/main/java/com/ibm/fhir/database/utils/db2/Db2Adapter.java

```diff
@@ -207,9 +245,9 @@ public void run() {
             // Not cool. This means that only some of the tables will have the partition assigned
             throw new DataAccessException("Tenant partition creation did not complete");
         }
-
+        
```

Comment:
please set your save actions to remove trailing spaces (per team convention)

Reply:
that's OK as long as all the files are clean, but it can cause big issues when dealing with merge conflicts as I recently experienced.

Reply:
yeah - we should cut it down when we all have it set.

Reply:
ok